pkg/version/version.go

```diff
@@ -92,6 +92,33 @@ func (i *Info) IsEqualOrNewer(inVer string) bool {
 	return true
 }
 
+// IsNewer returns true if the version in the Vdb is newer
+// than the given version
+func (i *Info) IsNewer(inVer string) bool {
```

Comment:
There is a lot of code duplication with  `IsEqualOrNewer`

Reply:
will reduce duplication